compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -234,7 +231,7 @@ protected Config() throws IOException {
                 new ObjCBlockPlugin(),
                 new AnnotationImplPlugin(),
                 new LambdaPlugin(),         
-                new ShadowFramePlugin()
+                new DebugInformationPlugin()
```

Comment:
@Tom-ski
it is utilizes facility from original RoboVM and there is no overhead at all during runtime. As there is no need to push shadow frame information. I've just added missing part (line number debug information) that connects method.c and ClassCompiler.java